coverage/index.html

```diff
@@ -0,0 +1,164 @@
+<!doctype html>
```

Comment:
Coverage folder should be ignored from the .gitignore file, don't push coverage folder.